packages/fsapp/src/beta-app/router/utils.ts

```diff
@@ -28,7 +28,8 @@ export const resolveRoutes = async ({
     for (const child of children) {
       // Replace Variables
       const searchPath = search.path?.replace(/:\w+(?=\/)?/, ':') ?? '';
-      const childPath = child.path?.replace(/:\w+(?=\/)?/, ':') ?? '';
+      const childPath = ('initialPath' in child ? child.initialPath : child.path)
+        ?.replace(/:\w+(?=\/)?/, ':') ?? '';
```

Comment:
Can we break this out into a utility function so that we don't need to disable `cyclomatic-complexity`?